test/src/state_machine/transitions.test.cpp

```diff
@@ -85,8 +88,23 @@ struct TransitionFunctionality : public ::testing::Test {
   int randomInRange(int, int);
 
  protected:
-  void SetUp() {}
-  void TearDown() {}
+  void SetUp()
+  {
+    fflush(stdout);
+    stdout_f     = dup(1);
+    tmp_stdout_f = open("/dev/null", O_WRONLY);
+    // Not on Unix, try Windows
```

Comment:
Iirc, the Makefiles do some detection of OS already. You could use that to add something like `-DWINDOWS` to CFLAGS. In code, you can then use `#ifdef WINDOWS` and such for changing the behaviour based on the OS. May or may not be neater - just an idea. 😄

Reply:
That's a pretty good idea!

Though, since we're not actually supporting Windows (as I've just learned), there's no real need for that anyways.